server/routes/index.js

```diff
@@ -135,6 +135,29 @@ router.get('/isLoggedIn', passport.authenticate('jwt', {session: false}), (req,
   res.json({message: "You have a fresh cookie!"});
 });
 
+router.patch('/changeTimeZone/', async (req, res, next) => {
```

Comment:
I put the route in index - I think it's too niche to go in it's own route file, and index typically deals with profile wide changes